frontend/public/components/utils/resource-log.tsx

```diff
@@ -402,6 +402,12 @@ export const ResourceLog: React.FC<ResourceLogProps> = ({
     true,
   );
 
+  React.useEffect(() => {
+    if (resource?.metadata?.annotations?.['console.openshift.io'] === 'wrap-log-lines') {
+      setWrapLines(true);
+    }
+  }, [resource, setWrapLines]);
```

Comment:
want an empty dependency array so the effect only runs once.

```suggestion
  // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```